pkg/nsx/services/securitypolicy/builder.go

```diff
@@ -177,9 +177,9 @@ func (service *SecurityPolicyService) buildTargetTags(obj *v1alpha1.SecurityPoli
 	return targetTags
 }
 
-func (service *SecurityPolicyService) buildBasicTags(obj *v1alpha1.SecurityPolicy) []model.Tag {
+func (service *Service) buildBasicTags(obj *v1alpha1.SecurityPolicy) []model.Tag {
 	uid := string(obj.UID)
-	clusterTag := getCluster(service)
```

Comment:
I suggest that we keep getCluster, the function is more helpful for future extension.  Also, we can consider put it as common function if necessary for all services.